qcore/constants.py

```diff
@@ -356,6 +393,8 @@ def check_mutually_exclusive_tasks(tasks: List["ProcessType"]):
         :return: A string containing any errors found during the check"""
         mutually_exclusive_tasks = (
             (ProcessType.BB, ProcessType.LF2BB, ProcessType.HF2BB),
+            # (ProcessType.GCMT_2_REL, ProcessType.NHM_2_REL),
```

Comment:
Is this comment needed

Reply:
Related to automated realisation generation, see above comments